imports/ui/containers/QuickGame1v1/AddPlayer/Game/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './styles.css';
+
+const Game = ({ text, onClick, buttonIcon, friendObject }) => (
```

Comment:
Remove unused props. Just do it. Do it as soon as you remove the prop from the component jsx.